pkg/apis/kappctrl/v1alpha1/status.go

```diff
@@ -25,6 +25,7 @@ const (
 )
 
 // TODO rename to Condition
+// +k8s:openapi-gen=true
```

Comment:
why do we need openapi-gen here?

Reply:
Packages use the same status as apps, so we need to have openapi schemas for the structs that make up a status in order for the apiserver to run.